lib/policy_machine/association.rb

```diff
@@ -31,23 +38,23 @@ def initialize(stored_user_attribute, stored_operation_set, stored_object_attrib
     # Returns true if the operation set of this association includes the given operation.
     #
     def includes_operation?(operation)
-      # Note:  operation_set.member? isn't calling PM::PolicyElement ==
-      operation_set.any?{ |op| op == operation }
+      # Note:  set_of_operation_objects.member? isn't calling PM::PolicyElement ==
+      set_of_operation_objects.any?{ |op| op == operation }
     end
 
     # Create an association given persisted policy elements
     #
-    def self.create(user_attribute_pe, operation_set, object_attribute_pe, policy_machine_uuid, pm_storage_adapter)
+    def self.create(user_attribute_pe, set_of_operation_objects, operation_set, object_attribute_pe, policy_machine_uuid, pm_storage_adapter)
       # argument errors for user_attribute_pe
       raise(ArgumentError, "user_attribute_pe must be a UserAttribute.") unless user_attribute_pe.is_a?(PM::UserAttribute)
       unless user_attribute_pe.policy_machine_uuid == policy_machine_uuid
         raise(ArgumentError, "user_attribute_pe must be in policy machine with uuid #{policy_machine_uuid}")
       end
 
-      # argument errors for operation_set
-      raise(ArgumentError, "operation_set must be a Set of Operations") unless operation_set.is_a?(Set)
-      raise(ArgumentError, "operation_set must not be empty") if operation_set.empty?
-      operation_set.each do |op|
+      # argument errors for set_of_operation_objects
+      raise(ArgumentError, "set_of_operation_objects must be a Set of Operations") unless set_of_operation_objects.is_a?(Set)
```

Comment:
don't wanna change all stylistically incorrect string formatting in the pm in this pr